phase2/grunt-drupal-tasks
# SOEOP-155 fix behat tests

# NOT READY FOR REVIEW

# Summary
_See ticket https://stanfordits.atlassian.net/browse/SOEOP-155:_ 
Fixing up broken behat tests.

# Needed By (Date)
- Soon

# Criticality
- Housekeeping
- Fixes broken stuff

# Steps to Test
- Checkout this branch
- Run behat tests: `grunt behat`
- There should be no failed tests 🤞 

# Affects 
- SoE (School of Engineering website)

# Associated Issues and/or People
## Related JIRA ticket(s)

https://stanfordits.atlassian.net/browse/SOEOP-155

## Related PRs

## More Information

## Folks to notify


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
